tsconfig.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "./node_modules/kcd-scripts/shared-tsconfig.json"
+  "extends": "./node_modules/kcd-scripts/shared-tsconfig.json",
+  "compilerOptions": {
+    "lib": ["es2019"]
```

Comment:
I'm not sure if this is the best way / place / solution, but without this addition type checking fails with the error `Property 'flat' does not exist on type 'string[]'. Do you need to change your target library? Try changing the `lib` compiler option to 'es2019' or later.`.